src/core/blockchain/types/client.ts

```diff
@@ -11,6 +11,11 @@ export interface IBlockInfo {
     hash?: string;
 }
 
+export interface IBalance {
+    total: BigNumber;
```

Comment:
There are 2 options here:
1. keep BigNumber as type here, but you have to be sure that BigNumber instances don't end up directly in redux. We would have problems on state rehydrate.
2. transform to string, redux will be fine, use BigNumber only where we need to use the amount to calculate something

Reply:
Before, value was as a BigNumber, I don't think it gets to redux as a BigNumber but I'll double check

Reply:
please do

Reply:
It's fine, they are all converted to string before